src/sentry/issues/endpoints/organization_group_search_views.py

```diff
@@ -110,6 +115,55 @@ def get(self, request: Request, organization: Organization) -> Response:
                     data={"detail": "You do not have access to any projects."},
                 )
 
+        visibility = request.GET.getlist("visibility", [])
```

Comment:
i believe getlist should return a list by default, so you shouldn't have to provide a second parameter.
https://docs.djangoproject.com/en/5.1/ref/request-response/#django.http.QueryDict.getlist
```suggestion
        visibility = request.GET.getlist("visibility")
```